rfcs/README.md

```diff
@@ -0,0 +1,108 @@
+
+Furthermore, the fact that a given RFC has been accepted and is 'active' implies nothing about what priority is assigned to its implementation, nor whether anybody is currently working on it.
+
+Modifications to active RFCs can be done in followup PRs. We strive to write each RFC in a manner that it will reflect the final design of the feature; but the nature of the process means that we cannot expect every merged RFC to actually reflect what the end result will be at the time of the next major release; therefore we try to keep each RFC document somewhat in sync with the language feature as planned, tracking such changes via followup pull requests to the document.
+
+## Reviewing RFCs
+
+[Reviewing RFCs]: #reviewing-rfcs
+
+While the RFC pull request is up, the team may schedule meetings with the author and/or relevant stakeholders to discuss the issues in greater detail. In such cases a summary from the meeting will be posted back to the RFC pull request.
+
+The team makes final decisions about RFCs after the benefits and drawbacks are well understood. These decisions can be made at any time, but the team will regularly issue decisions. When a decision is made, the RFC pull request will either be merged or closed. In either case, if the reasoning is not clear from the discussion in thread, the team will add a comment describing the rationale for the decision.
+
+## Implementing an RFC
+
+[Implementing an RFC]: #implementing-an-rfc
+
+Some accepted RFCs represent vital features that need to be implemented right away. Other accepted RFCs can represent features that can wait until some arbitrary developer feels like doing the work. Every accepted RFC will have an associated thread on [GitHub Discussions](https://github.com/blockprotocol/blockprotocol/discussions/categories/rfc) tracking its implementation in the Block Protocol repository; thus that associated discussion can be assigned a priority via the triage process that the team uses for all known-issues.
+
+The author of an RFC is not obligated to implement it. Of course, the RFC author (like any other developer) is welcome to post an implementation for review after the RFC has been accepted.
+
+If you are interested in working on the implementation for an "active" RFC, but cannot determine if someone else is already working on it, feel free to ask (e.g. by leaving a comment on the associated GitHub Discussion).
+
+## Inspiration
+
+[Inspiration]: #inspiration
+
+The Block Protocols's RFC process strongly owes its inspiration to the [Rust RFC process], and [React RFC process].
+
+[Rust RFC process]: https://github.com/rust-lang/rfcs
+[React RFC process]: https://github.com/reactjs/rfcs
+
+We're open to changing it as needed in response to feedback.
+
+[community Discord server]: https://blockprotocol.org/discord
+[developer discussion forum]: https://internals.rust-lang.org/
```

Comment:
I don't think this should be here (but if appropriate, our forum can be found at https://community.hash.ai/)

Reply:
I'm not sure how that's still here, I swear I did a ctrl + F for rust, good catch!